docs/contribute-add-feature.md

```diff
@@ -365,11 +389,15 @@ export const wordCountPlugin: Plugin<[], GenericParent, GenericParent> =
 This pattern, of "filling-in" the children of an existing node using a transform, is commonly used in the MyST ecosystem.
 
 If we compiled MyST with `npm run build`, we'd notice that this transform never runs. Like our earlier `word-count` role, we need to include the `wordCountTransform` in the set of MyST transforms that run during building. We can do this by adding a new _export_ line in `packages/myst-transforms/src/index.ts`:
-```typescript
+
+:::{code-block} typescript
+:filename: packages/myst-transforms/src/index.ts
 export { wordCountTransform, wordCountTransform } from './word-count.js';
-```
+:::
```

Comment:
I would suggest the `` ``` `` for anything that is actually code as some markdown formatters will change `_` and `*` aggressively.

Reply:
Could you elaborate? I want to keep the options, are you suggesting I use the fence-syntax with a directive?

Reply:
You can use the options with a fence directive! There are some docs here:

https://mystmd.org/guide/syntax-overview#directives

![image](https://github.com/executablebooks/mystmd/assets/913249/bd81a341-0323-4c42-a100-3d455ed218ac)

Reply:
Ah cool, I'm familiar with that.

I'll make the change.